pnpm-lock.yaml

```diff
@@ -7390,12 +7390,17 @@ packages:
 
   /async-listen@1.2.0:
     resolution: {integrity: sha512-CcEtRh/oc9Jc4uWeUwdpG/+Mb2YUHKmdaTf0gUr7Wa+bfp4xx70HOb3RuSTJMvqKNB1TkdTfjLdrcz2X4rkkZA==}
+    dev: true
 
   /async-listen@2.0.3:
```

Comment:
Any idea why we still have 2.0.3 in the dependency tree?

Reply:
```
@vercel/node@2.14.1 /Users/kikobeats/vercel/vercel/packages/node

dependencies:
async-listen 3.0.0
edge-runtime 2.1.4
└── async-listen 2.0.3
```

let me update in edge-runtime too

Reply:
arg I upgraded, but other changes are there; let me upgrade it as soon as we land edge-runtime@2.3.0; it's beta right now, but I don't want to introduce it here yet https://github.com/vercel/edge-runtime/pull/333